app/app.test.js

```diff
@@ -236,31 +252,45 @@ describe('POST',() => {
                             })
                          
                         })
-                        test('test with bad request', () => {
+                        test.only('test with bad request', () => {
                             
                             return request(app)
                             .delete(`/api/comments/cat`)
                             
                             .expect(400)
                             .then((response) => {
-                                expect(response.body.msg).toEqual("bad request")
+                                expect(response.body.msg).toEqual("invalid input type")
                             })
     
                            
                         })
-                        test('test with invalid id ', () => {
-                           
+                        test.only('test with invalid id ', () => {
                             return request(app)
-                            .delete(`/api/comments/9999999999`)
-                            
-                            .expect(400)
+                            .delete(`/api/comments/9999`)
+                            .expect(404)
                             .then((error) => {
-                                console.log(error.body)
                                 expect(error.body.msg).toEqual("invalid id")
                             })
     
                            
                         })
                     })
                 }) 
-            })
\ No newline at end of file
+            })
+describe('GET',() => {
+            describe('/api/users', () => {
+                test.only('return with an array of users objects, which have a "username", "name" and "avatar_url" property ', () => {
+                    return request(app)
+                    .get('/api/users')
+                    .expect(200)
+                    .then((response) => {
+                        console.log(response.body.users)
+                        response.body.users.forEach((user) => {
+                        expect(typeof user.username).toBe("string")
+                        expect(typeof user.name).toBe("string")
+                        expect(typeof user.avatar_url).toBe("string")  
+                       })
```

Comment:
whenever you do a loop to check an array of items in your testing you need to make sure you're not being returned a empty arr which gives this test a false positive. ALongside the forEach you need to make sure the arr has something in it